mindsdb/integrations/handlers/autosklearn_handler/autosklearn_handler.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional, Dict
+
+import dill
+import pandas as pd
+import autosklearn.classification as automl_classification
+import autosklearn.regression as automl_regression
+
+from .config import ClassificationConfig, RegressionConfig
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class AutoSklearnHandler(BaseMLEngine):
+    """
+    Integration with the Auto-Sklearn ML library.
+    """
+
+    name = 'autosklearn'
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[dict] = None) -> None:
+        config_args = {key: val for key, val in args['using'].items() if key != 'task'}
+
+        if args['using']['task'] == 'classification':
+            config = ClassificationConfig(**config_args)
+
+            model = automl_classification.AutoSklearnClassifier(**vars(config))
+
+        elif args['using']['task'] == 'regression':
+            config = RegressionConfig(**config_args)
+
+            model = automl_regression.AutoSklearnRegressor(**vars(config))
+
```

Comment:
I think we need an else here, just to raise an Exception when the task is not supported.

Reply:
Done!